.github/workflows/tii-mocap-pose.yaml

```diff
@@ -13,6 +15,18 @@ jobs:
 
       - uses: docker/setup-buildx-action@v1
 
+      - name: Set image tag format without suffix
+        run: |
+          echo "IMAGE_TAG_FORMAT=type=sha" >> $GITHUB_ENV
+        if: github.event_name == 'push'
+
+      - name: Set image tag format with suffix
+        # it is possible that run_number should be used instead run_attempt
+        # run_attempt is unique number on every run and run_attempt resets to 1 if re-build is not used
+        run: |
+          echo "IMAGE_TAG_FORMAT=type=sha,suffix=-${{ github.event.client_payload.run_attempt }}" >> $GITHUB_ENV
```

Comment:
The only minor idea I had about this review is that the client payload could've been named `image_sha_tag_suffix` so the name would not get outdated whether we use run_attempt or something else in the workflow dispatch of the fog-ros-baseimage repo.

But it was just a minor idea, not probably worth changing :)

Reply:
I have updated the name of the payload parameter in all the container where this work is being done.